src/tests/Microsoft.PowerFx.Core.Tests/BindingEngineTests.cs

```diff
@@ -220,10 +220,10 @@ public void CheckDottedBindError()
         {
             var config = new PowerFxConfig();
             var engine = new Engine(config);
-            var result = engine.Check("[1,2,3].foo");
+            var result = engine.Check("First([1,2,3]).foo");
```

Comment:
>First([1,2,3]).foo [](http://example.com/codeflow?start=39&length=18)

It may be good to have the existing test here, just validating the new error message.

Reply:
Adding a new test, keeping the existing one.